w3ctrace/state.go

```diff
@@ -19,13 +19,34 @@ type State []string
 func ParseState(traceStateValue string) (State, error) {
 	var state State
 
-	for _, st := range strings.SplitN(traceStateValue, ",", 32) {
-		st = strings.TrimSpace(st)
-		if st == "" {
-			continue
+	// comasLimit is used to prevent handling of malformed values that contain a lot of comas
+	const comasLimit = 256
+	rawEntries := strings.SplitN(traceStateValue, ",", comasLimit)
```

Comment:
```suggestion
	const maxEntries = 256
	rawEntries := strings.SplitN(traceStateValue, ",", maxEntries)
```

Reply:
This number is something just bigger than a reasonable amount of tracestate entries (in raw header). To prevent a situation when malicious user sent values like ",,,,,,,,,,,,,,,,,,,,,,,,,(and many comas after)" and system, will waste a CPU resource. If you think it is too much, we can use `Split` instead of `SplitN`. What do you think?